<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="minimumSize">
   <size>
    <width>669</width>
    <height>742</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>669</width>
    <height>742</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Filament Changer for Marlin 3D printers</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images Resources/icon_filament.png</normaloff>../Images Resources/icon_filament.png</iconset>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>651</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="lyt_archivo">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="FileLineEdit">
      <property name="text">
       <string>Choose file...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BrowseButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Preferences</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>651</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="LayerSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="LayerLabel">
       <property name="text">
        <string>Layer number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="EnabledCheckBox">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>660</y>
     <width>195</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="ExportBtn">
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ResetBtn">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
